glossary.html

```diff
@@ -16,29 +16,28 @@
           <main> 
              <h1>Acronym Glossary </h1>
                  <dl>
-                    <dt>HTML </dl> 
+                    <dt>HTML </dt> 
                         <dd> HyperText Markup Language</dd>  
-                    <dt> HTTP</dl> 
+                    <dt> HTTP</dt> 
                         <dd>HyperTest Transfer Protocol </dd>
-                    <dt>IP</dl> 
+                    <dt>IP</dt> 
                         <dd>Internet Protocol</dd>
-                     <dt> TCP</dl> 
+                     <dt> TCP</dt> 
                         <dd>Transmission Control Protocol </dd>
-                    <dt>URL </dl> 
+                    <dt>URL </dt> 
                         <dd> Uniform Resource Locator</dd>
                  </dl>
           </main>    
     <footer> 
         <hr> 
             <blockquote>
-            Everything stays, right where you left it,<br>
-            Everything stays, But it still changes<br>
-            Ever so slighty, daily and nightly, in<br>
-            little ways, everything stays.<br>
-            By: <strong> Rebeeca Sugar</strong>
+                Everything stays, right where you left it,<br>
+                Everything stays, But it still changes<br>
+                Ever so slighty, daily and nightly, in<br>
+                little ways, everything stays.<br>
+                <em>&mdash; <strong> Rebeeca Sugar</strong></em>
             </blockquote>
-                <p>
-                &copy 2019 <a href="mailto:carlin.linklater@student.saultcollege.ca"  >Carlin Linklater<a>            
+            <p>&copy; 2019 <a href="mailto:carlin.linklater@student.saultcollege.ca?subject=Message%20from%20CSD120%20Website">Carlin Linklater<a>            

```

Comment:
Copyright line should be in a `<p>` element
Unclosed character entity
Your email hyperlink doesn't set the subject